test/test_prims.py

```diff
@@ -752,6 +759,29 @@ def fn1(x):
             )
             self.assertFalse(includes_aten_to_copy)
 
+    @ops([op for op in op_db if op.name in {"permute"}], allowed_dtypes=(torch.float,))
```

Comment:
Let's extract the names that can be matched into a sequence and add a comment, or add OpInfo metadata that the operation supports varargs when used as a method.

I prefer adding OpInfo metadata, but we should do the hard work soon of improving the design metadata (with something like a signature) so these things can be inferred from a smaller amount of metadata

Reply:
Yeah, adding a flag to the metadata is probably a better way, but it's not always the method that has it (e.g. the ones/zeros/...).

Reply:
True but for `ones` and `zeros` can't we just add the varargs examples to their sample inputs?

Reply:
Not according to @SherlockNoMad https://github.com/pytorch/pytorch/pull/85117#discussion_r972591968

Reply:
It is not as straightforward as one might expect, but if you supplement

https://github.com/pytorch/pytorch/blob/9c036aa112b0a8fd9afb824d1fda058e2b66ba1d/torch/testing/_internal/common_methods_invocations.py#L822-L828

with
```python
    for size in sizes:
        yield SampleInput(size[0], args=size[1:], kwargs={'dtype': dtype, 'device': device})
```
it would seem that you actually get tests for varargs (tested by raising an exception in the ref temporarily).
That said, we can also test them a little bit in the varargs test.